packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -206,7 +206,17 @@ const componentFiles = async (name, scaffoldPath = '') => {
     DateTime: {
       displayFunction: 'timeTag',
     },
-    String: {
+    Int: {
+      name: 'NumberField',
+    },
+    Json: {
+      name: 'TextArea',
+      displayFunction: '',
```

Comment:
The generated code will be something like `{(post.jsonField)}`; notice the extra ( )
If we decide this is the route we want to take I'll make sure to fix this.